src/commands/runDraftTool/runDraftDockerfile.ts

```diff
@@ -0,0 +1,330 @@
+         throw Error('Language is undefined');
+      }
+      if (port === undefined) {
+         throw Error('Port is undefined');
+      }
+      if (sourceCodeFolder === undefined) {
+         throw Error('Source code folder is undefined');
+      }
+      if (version === undefined) {
+         throw Error('Version is undefined');
+      }
+
+      const configPath = buildCreateConfig(
+         language.id,
+         port,
+         '',
+         '',
+         version,
+         '',
+         ''
+      );
+      const command = buildCreateCommand(
+         sourceCodeFolder.fsPath,
+         'dockerfile',
+         configPath
+      );
+
+      const [success, err] = await runDraftCommand(command);
+      const isSuccess = err?.length === 0 && success?.length !== 0;
+      if (!isSuccess) {
+         throw Error(`Draft command failed: ${err}`);
+      }
+   }
+
+   public shouldExecute(wizardContext: WizardContext): boolean {
+      return true;
+   }
+}
+
+class ExecuteOpenDockerfiles extends AzureWizardExecuteStep<WizardContext> {
+   public priority: number = 2;
+   public async execute(
+      wizardContext: WizardContext,
+      progress: vscode.Progress<{
+         message?: string | undefined;
+         increment?: number | undefined;
+      }>
+   ): Promise<void> {
+      const dockerignorePath = getDockerignorePath(wizardContext);
+      const dockerPath = getDockerfilePath(wizardContext);
+
+      for (const filePath of [dockerignorePath, dockerPath]) {
+         const vscodePath = vscode.Uri.file(filePath);
+         await vscode.workspace
+            .openTextDocument(vscodePath)
+            .then((doc) =>
+               vscode.window.showTextDocument(doc, {preview: false})
+            );
+      }
+   }
+
+   public shouldExecute(wizardContext: WizardContext): boolean {
+      return true;
+   }
+}
+
+class ExecuteSaveState extends AzureWizardExecuteStep<WizardContext> {
+   public priority: number = 3;
+
+   constructor(private state: StateApi) {
+      super();
+   }
+
+   public async execute(
+      wizardContext: WizardContext,
+      progress: vscode.Progress<{
+         message?: string | undefined;
+         increment?: number | undefined;
+      }>
+   ): Promise<void> {
+      const {port} = wizardContext;
+      if (port !== undefined) {
+         this.state.setPort(port);
+      }
+
+      const dockerfilePath = getDockerfilePath(wizardContext);
+      this.state.setDockerfile(dockerfilePath);
+   }
+
+   public shouldExecute(wizardContext: WizardContext): boolean {
+      return true;
+   }
+}
+
+class ExecutePromptAcr extends AzureWizardExecuteStep<WizardContext> {
+   public priority: number = 4;
+
+   public async execute(
+      wizardContext: WizardContext,
+      progress: vscode.Progress<{
+         message?: string | undefined;
+         increment?: number | undefined;
+      }>
+   ): Promise<void> {
+      const buildAcrLink = 'Docs';
+      await vscode.window
+         .showInformationMessage(
```

Comment:
nit: would a message in the imperative be better? Something like 
The Dockerfile was created. Next, build the image on Azure Container Registry: (link)

Reply:
That is better. Swapped!